src/localization/localization.ts

```diff
@@ -35,6 +35,7 @@ const localization = {
                 "List of components in the codebase with their design counterparts—this is where the magic happens.",
             propNameInfo: "Custom name for the component to be displayed in Zeplin.",
             propPathInfo: "The file path for the component file.",
+            propZeplinIdsInfo: "Ids of the components in Zeplin to connect to, either one or more.",
```

Comment:
```suggestion
            propZeplinIdsInfo: "Component IDs in Zeplin to connect to, either one or more.",
```

Reply:
we referred to ids as "ids" at the middle of a sentence and "Ids" at the start of one. Let's keep it for consistency, wdyt?